README.md

```diff
@@ -28,3 +28,57 @@ The Presto Foundation maintains a [public calendar](https://calendar.google.com/
 The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers), and the current TSC chair is Nezih Yigitbasi ([@nezihyigitbasi](https://github.com/nezihyigitbasi)). At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project’s code repository
 
 https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md
+
+## Policies and procedures
+
+The Presto TSC is governed by the [Technical Charter](CHARTER.md).  The Charter provides a foundational structure for the TSC on topics such as its scope, how to make decisions, and how to make changes to itself.  At the same time, it grants the TSC a high degree of freedom when determining how to implement the policies of the Presto Foundation.
+
+The following policies and procedures have been adopted by the TSC.
+
+### Making decisions
+
+Per the [charter](CHARTER.md), wherever possible the TSC will attempt to make decisions by consensus.  In circumstances where consensus is not possible or if a vote is explicitly required, a majority (or higher, if required by the governance) of TSC voting members must approve in order for the action to proceed.  Votes will be taken over email, and documented in the next meeting.
```

Comment:
Just trying to understand better: Does each TSC member have one vote or each company have one vote? Sorry if this is clarified somewhere else.